resource/supported.go

```diff
@@ -73,6 +75,7 @@ const (
 	S3Bucket            TerraformResourceType = "aws_s3_bucket"
 	SecurityGroup       TerraformResourceType = "aws_security_group"
 	Subnet              TerraformResourceType = "aws_subnet"
+	Trail               TerraformResourceType = "aws_cloudtrail_trail"
```

Comment:
```suggestion
	CloudTrail               TerraformResourceType = "aws_cloudtrail"
```

Reply:
Adjusted but why did you derive `CloudTrail` from? Technically it's a service and within it you can have multiple trails but yeah, no big deal.

Reply:
Correct, it's a service, but in Terraform each trail is defined by a resource type called `aws_cloudtrail` (see here: https://www.terraform.io/docs/providers/aws/r/cloudtrail.html)